src/utils/makeReactNativeConfig.js

```diff
@@ -197,6 +213,10 @@ const getDefaultConfig = (
     mainFields: ['react-native', 'browser', 'main'],
     extensions: [`.${platform}.js`, '.native.js', '.js'],
   },
+  /**
+   * Set target to webworker as it's closer to RN environment than `web`.
+   */
+  target: 'webworker',
```

Comment:
I've check all templates for `web` and `webworker` and they seems to differ only in terms of script/files loading.